lab_1_classify_by_unigrams/main.py

```diff
@@ -65,6 +91,15 @@ def calculate_mse(predicted: list, actual: list) -> float | None:
     In case of corrupt input arguments, None is returned
     """
 
+    if len(actual) != len(predicted):
+        return None
+    summation = 0
```

Comment:
Is `summations` eng word?

Reply:
Yeah… But now I guess summ will be better